src/wiki/static/wiki/bootstrap/css/wiki-bootstrap.css

```diff
@@ -1081,7 +1081,7 @@ body {
   font-size: 14px;
   line-height: 1.42857143;
   color: #333333;
-  background-color: #fff;
+  background-color: #ffffff;
```

Comment:
Please reward changes in this file. They all appear to change nothing.

Reply:
There is one new ".article-edit-title-link", and because of that I had to
regenerate this file with lessc, which than unfortunately created the other changes.